tests/e2e/workloads/pgsql/test_pgsql.py

```diff
@@ -38,11 +43,25 @@ def test_sql_workload_simple(self, pgsql):
             replicas=3, clients=3, transactions=600
         )
 
+        # Start measuring time
+        start_time = datetime.now()
+
+        # Check worker node utilization (adm_top)
+        get_node_resource_utilization_from_adm_top(node_type='worker', print_table=True)
+
+        # Check worker node utilization (oc_describe)
+        get_node_resource_utilization_from_oc_describe(node_type='worker', print_table=True)
+
         # Wait for pg_bench pod to initialized and complete
         pgsql.wait_for_pgbench_status(status=constants.STATUS_COMPLETED)
 
+        # Calculate the time from running state to completed state
+        end_time = datetime.now()
+        diff_time = end_time - start_time
+        log.info(f"\npgbench pod reached to completed state after {diff_time.seconds} seconds\n")
+
         # Get pgbench pods
         pgbench_pods = pgsql.get_pgbench_pods()
 
         # Validate pgbench run and parse logs
-        pgsql.validate_pgbench_run(pgbench_pods)
+        pgsql.get_pgbech_pod_status_table(pgbench_pods)
```

Comment:
You have removed validate_pgbench_run().
 validate_pgbench_run() returns pg_output in list which is used for exporting the data to gsheet. Removing this will break that.
Can we move this generation of pretty table to validate_pgbench_run()?

Please let me know your thoughts.